bot/exts/fun/latex.py

```diff
@@ -105,6 +113,21 @@ async def _upload_to_pastebin(self, text: str) -> str | None:
         except web.HTTPClientError as e:
             log.info("Error when uploading latex output to pastebin. %s", e)
 
+    async def _prepare_error_embed(self, err: Exception | None) -> discord.Embed:
```

Comment:
The type hinting on this one isn't quite valid since you're only catching the two custom latex exceptions. 

```suggestion
    async def _prepare_error_embed(self, err: InvalidLatexError | LatexServerError |  None) -> discord.Embed:
```

Reply:
Indeed, my bad :')